features/page-breaks.yml.dist

```diff
@@ -104,6 +102,14 @@ compat_features:
   #   firefox_android: "92"
   - css.properties.break-inside.paged_context.avoid-page
 
+  # baseline: false
+  # support:
+  #   chrome: "50"
+  #   chrome_android: "50"
+  #   edge: "79"
+  - css.properties.break-after.avoid
+  - css.properties.break-before.avoid
```

Comment:
These are partial in Firefox and Safari. See https://github.com/mdn/browser-compat-data/pull/27085.